src/applications/hca/config/chapters/insuranceInformation/general.js

```diff
@@ -1,53 +1,107 @@
+import React from 'react';
 import merge from 'lodash/merge';
-import get from 'platform/utilities/data/get';
 import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
 
 import InsuranceProviderView from '../../../components/InsuranceProviderView';
 
+import {
+  healthInsuranceCoverageQuestionDescription,
+  hasTricareWhatIsMyPolicyNumberDescription,
+  healthInsuranceDescription,
+} from '../../../helpers';
+
 const { provider } = fullSchemaHca.definitions;
 const { isCoveredByHealthInsurance } = fullSchemaHca.properties;
 
 export default {
   uiSchema: {
-    'ui:title': 'Other coverage',
+    'ui:description': healthInsuranceDescription,
     isCoveredByHealthInsurance: {
-      'ui:title':
-        'Are you covered by health insurance? (Including coverage through a spouse or another person)',
+      'ui:title': 'Do you have health insurance coverage?',
+      'ui:description': healthInsuranceCoverageQuestionDescription,
       'ui:widget': 'yesNo',
     },
     providers: {
       'ui:options': {
-        itemName: 'Insurance Policy',
         expandUnder: 'isCoveredByHealthInsurance',
+        itemName: 'insurance policy',
+        hideTitle: true,
         viewField: InsuranceProviderView,
       },
       'ui:errorMessages': {
         minItems: 'You need to at least one provider.',
       },
       items: {
         insuranceName: {
-          'ui:title': 'Name of provider',
+          'ui:title': 'Name of insurance provider',
+          'ui:autofocus': true,
```

Comment:
I'm not sure if `autofocus` is okay to use here. We generally focus on the `h1` or breadcrumbs when going to a new page in a form, but I would definitely ask an accessibility expert about it.

Reply:
removed, meant to remove before commit, thanks for all your feedback, I'm new at Ad Hoc, this is my first pull request.